.clang-format

```diff
@@ -0,0 +1,29 @@
+BasedOnStyle: llvm
```

Comment:
Should be based on google's style guide.

Reply:
I prefer Google style myself, but this code was written a while ago.

If you just set it as Google style, with longer lines, and don't reformat horizontal alignment, I think that would be good.

Reply:
> don't reformat horizontal alignment

Not sure if this is possible to tell to clang-format.